### ISSUE TITLE
Transfer this repo to @IllinoisLegalAidOnline, archive it, or ?

### ISSUE THREAD

**Issue by samglover** [samglover/docassemble-AppearanceEfile#11]
This interview—or a version of it—appears to be [live on ILAO](https://easyforms.illinoislegalaid.org/run/AppearanceEfile/appearance/#/1), but not on CourtFormsOnline. Also, [ILAO has an appearance repo](https://github.com/IllinoisLegalAidOnline/docassemble-Appearance) that may be the authoritative repo for this interview, but it's not clear because ILAO has [two appearance interviews](https://www.illinoislegalaid.org/form-search?keyword=appearance) on its website. There is one for appearance, and one for appearance with e-filing.

So either this is the authoritative repo and it should be transferred to @IllinoisLegalAidOnline, or this is not and it should be archived.

@nonprofittechy @mnewsted

**Comment by mnewsted** [comment]
I believe the Suffolk version can be archived.  The ILAO repo for
AppearanceEfile is the one used in production. I believe the Suffolk
version was used during early development, or maybe subsequent development.

You are correct that ILAO has 2 separate Appearance programs. One just
makes the forms and one makes the forms and e-files.

Please let me know if you have any questions. Thanks!

*Matt Newsted (he/him/his) | Associate Director of Legal Tech | *
*Illinois Legal Aid Online <https://www.illinoislegalaid.org/>120 S LaSalle
St, Suite 900 | Chicago, IL 60603 | 312.977.9047 ext 127 <(312)%20977-9047>*

*ILAO proudly supports Pride Month with these LGBTQIA+ centered resources
<https://www.illinoislegalaid.org/legal-information/population/member-lgbtqia-community>.*

**Comment by samglover** [comment]
Thanks @mnewsted! Archiving.

**Comment by BryceStevenWilley** [comment]
Sorry to unarchive, but I just saw this. This repo **is** the one being used and is definitely the one that has been most recently developed on.

Not sure what you're talking about Matt, since ILAO doesn't have an AppearanceEfile repo, just an Apearance one. So I think this repo should be transferred to ILAO, if that's what we want.

**Comment by mnewsted** [comment]
Thanks Bryce. Sorry, I forgot the dev process. Let me know if we should
transfer them all over to ILAO so development can continue.

*Matt Newsted (he/him/his) | Associate Director of Legal Tech | *
*Illinois Legal Aid Online <https://www.illinoislegalaid.org/>120 S LaSalle
St, Suite 900 | Chicago, IL 60603 | 312.977.9047 ext 127 <(312)%20977-9047>*

*ILAO proudly supports Pride Month with these LGBTQIA+ centered resources
<https://www.illinoislegalaid.org/legal-information/population/member-lgbtqia-community>.*

**Comment by samglover** [comment]
Thanks for catching that, @BryceStevenWilley! @mnewsted I will initiate a transfer so that you can manage the repo.

**Comment by samglover** [comment]
@mnewsted Transferring is a little clunky. I can't transfer it directly to @IllinoisLegalAidOnline. I have to transfer it to myself, then to you, and then you can transfer it to the @IllinoisLegalAidOnline organization.

Keep an eye out for that!

**Comment by samglover** [comment]
@BryceStevenWilley or @nonprofittechy Can you think of any implications I might be missing if this repo is transferred from @SuffolkLITLab to @IllinoisLegalAidOnline? It seemed obvious to me that ILAO should control its own interviews, which is why I initiated the transfer.

**Comment by BryceStevenWilley** [comment]
No, not any downsides I can think of. Github is pretty good at redirecting all of the links and repository URLs, and a quick search says there are only issues when there are private repos involved, which we don't have here.

I did also find that you should be able to transfer the repository directly to @IllinoisLegalAidOnline if you have the permissions to make a new repo there. So you could add @mnewsted as an admin on this repo (which he should have anyway, IMO) and let him transfer it directly [following these instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization).

**Comment by BryceStevenWilley** [comment]
I see that this was done! In case @mnewsted comes across this issue, it should be able to be closed, as the repo is now under ILAO's stewardship!
